src/harness/scripts/create-airgap-bundle.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+images="harness-airgapped-images.tar.gz"
```

Comment:
Can we make extension of file is .tgz?

Reply:
Is there any benefit of using .tgz instead of .tar.gz? Linux/Unix based systems mostly use .tar.gz